packages/dx-react-grid-material-ui/src/templates/column-chooser/item.jsx

```diff
@@ -5,36 +5,38 @@ import Checkbox from 'material-ui/Checkbox';
 
 export const Item = ({
   item: { column, hidden },
-  onToggle,
+  disabled, onToggle,
   ...restProps
 }) => (
   <ListItem
     key={column.name}
-    button
+    button={!disabled}
     onClick={onToggle}
```

Comment:
Is `onClick` available while the `disabled` property is true?

Reply:
`onToggle` contains nothing in this case.

Reply:
I mean that if button=false the ListItem is not available for click event and we should not write `disable ? onToggle=undefined : ...`

Reply:
I think that we should delegate disabled state handling to the Item components.